### ISSUE TITLE
OSDs restarted outside play scope

### ISSUE THREAD

**Issue by Logan2211** [ceph/ceph-ansible#1457]
When running ceph-common with a `--limit` and a restart is notified on any of the ceph-ansible services, the new restart method has no concept of scoping its restarts to hosts targeted by the playbook run. Instead it will try to restart OSDs on every host in the group, not just the ones currently being operated on. This is really a huge regression because it completely removes the control of OSD restarts from the operator's control.

The conditionals in the file that control the host selection for restarts (ie. https://github.com/ceph/ceph-ansible/blob/e8e4245e434b5405d489714a31557252a9ab186d/roles/ceph-common/handlers/main.yml#L43) should be modified to reference play_hosts ex.

```
  when:
    - osd_group_name in group_names
    - item in play_hosts
```

**Comment by Logan2211** [comment]
ps. love the host-serialized restarts though :)

**Comment by leseb** [comment]
Thanks for reporting this @Logan2211 , we will try to address this in https://github.com/ceph/ceph-ansible/pull/1455

**Comment by andrewschoen** [comment]
Fixed with #1455
